cmake/utils/CheckCXXFeatures.cmake

```diff
@@ -85,31 +85,32 @@ macro(CHECK_CXX_STANDARD_LIBRARY)
     if (NOT HAVE_NAMESPACES_DONE)
         CHECK_CXX_FEATURE(HAVE_NAMESPACES namespaces.cpp "implements namespaces")
     endif()
-    # CHECK_CXX_FEATURE(HAVE_COMPLEX                       have_complex.cpp                  "has complex<T>")
-    # CHECK_CXX_FEATURE(HAVE_COMPLEX_FCNS                  have_complex_fcns.cpp             "has standard complex<T> functions")
-    # CHECK_HEADER(HAVE_CSTRING cstring)
-    # CHECK_CXX_FEATURE(HAVE_NUMERIC_LIMITS                have_numeric_limits.cpp           "has numeric_limits<T>")
-    # CHECK_CXX_FEATURE(HAVE_CLIMITS                       have_climits.cpp                  "has <climits> header")
-    # CHECK_CXX_FEATURE(HAVE_VALARRAY                      have_valarray.cpp                 "has valarray<T>")
-    # CHECK_CXX_FEATURE(HAVE_COMPLEX_MATH1                 have_complex_math1.cpp            "has complex math functions")
-    # CHECK_CXX_FEATURE(HAVE_COMPLEX_MATH2                 have_complex_math2.cpp            "has more complex math functions")
-    # CHECK_CXX_FEATURE(HAVE_IEEE_MATH                     have_ieee_math.cpp                "supports IEEE math library")
-    # CHECK_CXX_FEATURE(HAVE_SYSTEM_V_MATH                 have_system_v_math.cpp            "supports System V math library")
-    # CHECK_CXX_FEATURE(HAVE_MATH_FN_IN_NAMESPACE_STD      math_fn_in_namespace_std.cpp      "has C math functions in namespace std")
-    # CHECK_CXX_FEATURE(HAVE_MATH_ABSINT_IN_NAMESPACE_STD  math_absint_in_namespace_std.cpp  "has C math abs(integer type) in namespace std")
-    # CHECK_CXX_FEATURE(HAVE_COMPLEX_MATH_IN_NAMESPACE_STD complex_math_in_namespace_std.cpp "supports complex math functions are in namespace std")
-    # CHECK_CXX_FEATURE(HAVE_ISNAN_IN_NAMESPACE_STD        isnan_in_namespace_std.cpp        "has isnan function in namespace std")
```

Comment:
FYI @papadop  these had been commented out as it was not used and it was increasing cmake bulid time for nothing.

Reply:
Yes pre-approved. Maybe you can integrate it with your old CMake removal. I already have merged this branch....

Reply:
see https://github.com/openmeeg/openmeeg/pull/403